vendor/github.com/cortexproject/cortex/pkg/ruler/ruler.go

```diff
@@ -167,7 +167,12 @@ func (cfg *Config) RegisterFlags(f *flag.FlagSet) {
 	f.StringVar(&cfg.ShardingStrategy, "ruler.sharding-strategy", util.ShardingStrategyDefault, fmt.Sprintf("The sharding strategy to use. Supported values are: %s.", strings.Join(supportedShardingStrategies, ", ")))
 	f.DurationVar(&cfg.FlushCheckPeriod, "ruler.flush-period", 1*time.Minute, "Period with which to attempt to flush rule groups.")
 	f.StringVar(&cfg.RulePath, "ruler.rule-path", "/rules", "file path to store temporary rule files for the prometheus rule managers")
-	f.BoolVar(&cfg.EnableAPI, "experimental.ruler.enable-api", false, "Enable the ruler api")
+
+	// Deprecated: this API is no longer experimental, but we'll register it under both flags for now.
```

Comment:
Uh, I guess I edited the cortex vendor 🤕 . I'll clean this up to work on the Loki wrapper instead (or should I just upstream this into cortex 🤔 ?)